ROADMAP.md

```diff
@@ -8,51 +8,27 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
 
-## Rook 1.1
+## Rook 1.2
 
+* Integration
+  * Speed up integration tests [#1218](https://github.com/rook/rook/issues/1218)
 * Ceph
-  * Remove support for Ceph Luminous
-  * Stable release of Ceph-CSI plug-in (feature parity with FlexVolume)
-  * Connect to an external Ceph cluster [#2175](https://github.com/rook/rook/issues/2175)
-  * Mon placement respects failure domains [#2603](https://github.com/rook/rook/issues/2603)
-  * User-modifiable configuration at runtime [#2470](https://github.com/rook/rook/pull/2470/files)
-  * Document a safe shutdown procedure [#2517](https://github.com/rook/rook/issues/2517)
-  * Support for dynamic provisioning of buckets [#1705](https://github.com/rook/rook/issues/1705)
-  * K8s upgrade support based on dynamic PDBs and MDBs [#3577](https://github.com/rook/rook/issues/3577)
-  * Upgrades will wait for healthy Ceph state before proceeding with each daemon restart [#2889](https://github.com/rook/rook/issues/2889)
-  * Run on arbitrary PVs (e.g. local storage) as an alternative to host path
-    * Support for more dynamic clusters such as GKE [#2107](https://github.com/rook/rook/issues/2107)
-    * Mons and OSDs can be configured to consume PVs for persistent storage [#919](https://github.com/rook/rook/issues/919)
-  * OSDs
-    * Allow Ceph disk selection by full path [#1228](https://github.com/rook/rook/issues/1228)
-    * Leverage node labels for CRUSH locations [#1366](https://github.com/rook/rook/issues/1366)
+  * Support for multi-networking (Multus) to provide more secure storage configuration [#4077](https://github.com/rook/rook/pull/4077)
```

Comment:
I don't think we can have anything for 1.2, at this point we are a bit blocked, perhaps we can finalize the design only?

Reply:
For now I'll keep the Multus line in the future improvements section instead of a specific release. There are enough questions on the design that it isn't clear when we can commit it to a release. What about the ipv4/ipv6 dual stack issue?

Reply:
It's alpha in 1.16, I can do it for Ceph in 1.2. Let's keep it in!